### DISCUSSION TITLE
advise to add a .string() function

### DISCUSSION THREAD

**Discussion by Nyx2022** [projectdiscovery/rawhttp#49] [General]
now,rawhttp only has .body to get response body,however, this will occur "i/o timeout" problem  in some cases such as in proxy environt,so i advise rawhttp to add a .string() to get response body directly

**Comment by Mzack9999** [comment]
@Nyx2022 `rawhttp` attempts to return the data as a valid compatible `net.Http` struct so that it can be used as a drop-in replacement of `net/http` where complete control is needed. I think the proxy you are testing against might reject malformed requests. Unless you are doing it for security purposes, or require full control of the parts of the request, probably using the standard library would make proxy interactions much more straightforward.
